src/java.xml/share/classes/javax/xml/catalog/CatalogManager.java

```diff
@@ -85,6 +88,34 @@ public static CatalogResolver catalogResolver(Catalog catalog) {
         return new CatalogResolverImpl(catalog);
     }
 
+    /**
+     * Creates an instance of a {@code CatalogResolver} using the specified catalog
+     * and the value of the {@link CatalogFeatures.Feature#RESOLVE RESOLVE} property.
+     *
+     * @param catalog the catalog instance
+     * @param resolve the value of the {@link CatalogFeatures.Feature#RESOLVE RESOLVE}
+     * property that overrides the previous setting used for creating the {@code catalog}
+     * object. The supported values are: {@code strict}, {@code continue},
+     * and {@code ignore}. {@code null} may be specified to indicate that the
+     * {@code catalog} object's current {@link CatalogFeatures.Feature#RESOLVE RESOLVE}
+     * value remains unchanged.
```

Comment:
Supported values explanation and the behavior on `null` string may be moved into the method description body, as it explains how this method is supposed to work.